tike/ptycho.py

```diff
@@ -215,8 +216,8 @@ def combine_grids(grids, v, h,
 
     # SHARED
```

Comment:
Wouldn't this increase the complexity of the code and make it harder to read?

Reply:
If only one or two lines of code, maybe yes, but if more than >5 lines then it is debatable. It may desire a better docs about what is going there.

Reply:
Yeah. Duplicate code was removed, but the net change in lines is zero because documentation.